doc/source/tutorial-quickstart-jax.rst

```diff
@@ -248,41 +248,101 @@ We included type annotations to give you a better understanding of the data type
                 {"loss": float(loss)},
             )
 
-Having defined the federation process, we can run it.
+Next, we create a client function that returns instances of :code:`FlowerClient` on-demand when called:
 
 .. code-block:: python
 
-    def main() -> None:
-        """Load data, start MNISTClient."""
+    def client_fn(cid: str):
+        return FlowerClient().to_client()
 
-        # Load data
-        train_x, train_y, test_x, test_y = jax_training.load_data()
-        grad_fn = jax.grad(jax_training.loss_fn)
+Finally, we create a :code:`ClientApp()` object that uses this client function:
 
-        # Load model (from centralized training) and initialize parameters
-        model_shape = train_x.shape[1:]
-        params = jax_training.load_model(model_shape)
+.. code-block:: python
 
-        # Start Flower client
-        client = FlowerClient(params, grad_fn, train_x, train_y, test_x, test_y)
-        fl.client.start_client(server_address="0.0.0.0:8080", client=client.to_client())
+    app = ClientApp(client_fn=client_fn)
+
+That's it for the client. We only have to implement :code:`Client` or :code:`NumPyClient`, create a :code:`ClientApp`, and pass the client function to it. If we implement a client of type :code:`NumPyClient` we'll need to first call its :code:`to_client()` method.
 
-    if __name__ == "__main__":
-        main()
 
+Flower Server
+^^^^^^^^^^^^^
 
-And that's it. You can now open two additional terminal windows and run
+For simple workloads, we create a :code:`ServerApp` and leave all the
+configuration possibilities at their default values. In a file named
+:code:`server.py`, import Flower and create a :code:`ServerApp`:
 
 .. code-block:: python
 
-    python3 client.py
+    from flwr.server import ServerApp
+
+    app = ServerApp()
+
+
+Train the model, federated!
+---------------------------
+
+With both :code:`ClientApps` and :code:`ServerApp` ready, we can now run everything and see federated
+learning in action. First, we run the :code:`flower-superlink` command in one terminal to start the infrastructure. This step only needs to be run once.
+
+.. admonition:: Note
+    :class: note
+
+    In this example, the :code:`--insecure` command line argument starts Flower without HTTPS and is only used for prototyping. To run with HTTPS, we instead use the argument :code:`--certificates` and pass the paths to the certificates. Please refer to `Flower CLI reference <ref-api-cli.html>`_ for implementation details.
```

Comment:
We now use `--ssl-ca-certfile`, `--ssl-certfile`, and `--ssl-keyfile` instead of `--certificate`.

Reply:
Fixed!